public/tpl/header.html

```diff
@@ -44,6 +44,9 @@
                 <li class="active">
                     <a href="/">Home</a>
                 </li>
+                <li ng-show="settings.useRichList">
```

Comment:
This is what I meant with the `config.settings.useRichList` comment for your environment variable. Personally, I think having the `useRichList` option inside `settings` is the best approach. Just need to resolve the other one in `tools/sync.js` line 463

Reply:
there are two kind of switch, one is for the `tools/sync.js` backend (`config.userRichList`), the other is frontend switch (`config.settings.useRichList`),
sorry for less documentaton anyway.

Reply:
Hrm, this is all fine, but maybe it's best to change the variable name of one of them so that it's less confusing in the documentation. Whichever one is less work. For example, if it takes less work to change the front-end switch, I'd prefer to give it a name like `showRichList`

Reply:
thank you for your review! I will update my PR ASAP. :)